.phpactor.json

```diff
@@ -0,0 +1,4 @@
+{
+    "language_server_phpstan.enabled": false
+}
+
```

Comment:
After spending time with these in our repos, I think `.phpactor.json` should just be .gitignored. We may not always be using the same config.